fastpass/src/main/scala/scala/meta/internal/fastpass/pantsbuild/BloopPants.scala

```diff
@@ -165,6 +165,7 @@ object BloopPants {
       s"--${noInternalSources}export-fastpass-sources",
       s"--${no3rdPartySources}export-fastpass-libraries-sources",
       s"--export-fastpass-output-file=$outputFile",
+      s"--export-fastpass-allow-global-excludes=False",
```

Comment:
Is this flag necessary? There are no other clients using `export-fastpass` so I wonder if we can make this flag enabled by default 🤔

Reply:
We can set this from `pants.ini` instead. I personally prefer to have the tool "configure itself", but I'm happy to change it if you think that'd be better.